packages/system-data/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	"extends": "@directus/tsconfig/base",
+	"compilerOptions": {
+		"module": "NodeNext",
+		"lib": ["ES2020", "DOM", "DOM.Iterable"],
```

Comment:
Do we need the DOM?

Reply:
We'll see once the tsup config is complete